### ISSUE TITLE
Hitting ESC while focused on Source Pane also impacts Console Pane

### ISSUE THREAD

**Issue by dantonnoriega** [rstudio/rstudio#8254]
### System details

    RStudio Edition : Desktop
    RStudio Version : Version 1.4.953
    OS Version      :  macOS Catalina 10.15.7
    R Version       :  R version 3.6.3 Patched (2020-04-28 r78687)

### Steps to reproduce the problem

Hitting `Esc` key while in Editor / Source window always affects Console.

### Describe the problem in detail

While editing and running a job, I find that if I hit the ESC key when focused on the editor, it also impacts the console. I don't feel like this was the case before. Super frustrating when running a long job, hitting ESC to say deselect a quick add next, only to find that your job was also cancelled.

(I personally hit ESC frequently when editing code.)

### Describe the behavior you expected

I expect that hitting ESC should only impact the focused panel e.g. if focused on editor/source, ESC only affects the editor and that the console be isolated.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- ~~[ ] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).~~
- ~~[ ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.~~

**Comment by kevinushey** [comment]
I see the same behavior in older releases of RStudio (even back in 1.0) so this seems like it's been the case for a while. Are you sure behavior has changed in this regard? Can you elaborate a bit more on when exactly you're seeing this behavior?

If you are willing, older releases of RStudio are available here:

https://rstudio.com/products/rstudio/older-versions/

and you can check whether the behavior has indeed changed as you describe.

**Comment by dantonnoriega** [comment]
Yeah, I think I'm wrong here. I suppose I just rarely had instances where I had long running code and where I was hitting ESC frequently in the past that it never presented a problem.

I also only recently moved to RStudio from ST3 (basically once ya'll matched ST3 keyboard shortcuts, I opted to switch).

Perhaps I should instead file this as a feature request? I don't understand how folks who, for example, use VIM mode in RStudio don't end up constantly cancelling jobs in the console.
